IPBT Webservice

mileo opened the pull request.

mileo committed:
[NEW] IBPT webservices

mileo committed:
[NEW] View improve l10n_br_tax.estimate

mileo committed:
[NEW] IPBT webservices request on account_fiscal_position

mileo committed:
[NEW] ResConfig settings for l10n_br_account, [FIX] IPBT [NEW] WS token

squash

renatonlima commented:
Vou começar a testar e fazer o review

mileo commented:
Show! Tinha que aprovar os PEP8 e os testes novos antes pra ficar sempre verde.
Pq tem hora q to desenvolvendo com base na 8.0 e esta dando problema.

Por hoje chega! 

Se conseguir tirar alguns campos extras que vc colocou vale a pena, podemos criar uma ação automatizada pra ele consultar as NCM que tem produtos 1x por semana. 

[]s

danimaribeiro commented on l10n_br_account_product/models/account_product_fiscal_classification.py:
```diff
@@ -215,6 +220,38 @@ def _compute_taxes(self):
+        config = DeOlhoNoImposto(company.ipbt_token,
+                                 punctuation_rm(company.cnpj_cpf),
+                                 company.state_id.code)
+        tax_estimate = self.env['l10n_br_tax.estimate']
```
Can you iterate over self? it is a multi method.
This way would be easy to add a context button in tree view, to execute over a bunch of models together.

mileo replied:
Nice idea! I will work on it.

mileo committed:
[NEW] Iterate on ipbt

mileo committed:
[FIX] l10n_br_tax.estimate relation

Conflicts:
	l10n_br_account_product/models/account_product_fiscal_classification.py

mileo committed:
[FIX] l10n_br_tax_estimate update search domain

mileo commented:
@renatonlima Rebased and fixed

mileo commented:
Your opinion regarding the use only in asset acquisition transactions was correct.

mileo committed:
[FIX] Add security rule

squash

mileo commented:
@renatonlima travis ok, but i can't insert a personal token to increase coverage.
@rvalyi

Daniel-Furcht commented:
Funcionou perfeitamente !!!
Preveste alguma forma de calcular automaticamente e passar o valor para o produto na NFe?
